tests/baselines/reference/findAllReferencesLinkTag1.baseline.jsonc

```diff
@@ -0,0 +1,857 @@
+// === /tests/cases/fourslash/findAllReferencesLinkTag1.ts ===
+// class C {
+//     [|m|]() { }
```

Comment:
Seems like the `/*FIND ALL REFS*/` comment marker didn’t render on `1` or `2` (`m` or `n`) 🤔

Reply:
I'm not familiar enough with this baseline format. Because it all works in my editor I assumed it the baselines were correct too.

Reply:
I think this is just a bug in the baseline text generation. I can try to take a look.

Reply:
Can you cherry-pick / copy and paste this? https://github.com/microsoft/TypeScript/compare/link-tag-method-name-resolution...andrewbranch:link-tag-method-name-resolution?expand=1